examples/grpc-kotlin/src/main/kotlin/example/armeria/grpc/kotlin/HelloMessageFactory.kt

```diff
@@ -0,0 +1,14 @@
+package example.armeria.grpc.kotlin
+
+import example.armeria.grpc.kotlin.Hello.HelloReply
+import example.armeria.grpc.kotlin.Hello.HelloRequest
+
+class HelloMessageFactory {
```

Comment:
How about using private methods like the other examples?

https://github.com/line/armeria/blob/master/examples/grpc-reactor/src/main/java/example/armeria/grpc/reactor/HelloServiceImpl.java#L104

Unless that's a total antipattern in Kotlin, I think it's better for the examples to be as close to each other as possible for comparison

Reply:
> I think it's better for the examples to be as close to each other as possible for comparison

That makes sense. 
@heowc Sorry for the interrupting multiple times. 😅 I think @anuraaga ‘s suggestion is better than mine.

Reply:
Fine. 😄I will reflect on the code soon.